app/models/manageiq/providers/redhat/inventory/persister/definitions/infra_group/vms_collections.rb

```diff
@@ -81,14 +81,6 @@ def add_operating_systems
 
   # group :vms
   def add_vm_and_template_ems_custom_fields
-    add_collection(infra, :vm_and_template_ems_custom_fields, {}, {:auto_inventory_attributes => false}) do |builder|
-      builder.add_properties(
-        :model_class => ::CustomAttribute,
-        :manager_ref => %i(name)
-      )
-      builder.add_properties(:arel => CustomAttribute.joins("INNER JOIN vms ON vms.id = custom_attributes.resource_id").where(:vms => { :ems_ref => manager_refs })) if targeted?
-
-      builder.add_inventory_attributes(%i(section name value source resource))
-    end
+    add_collection(infra, :vm_and_template_ems_custom_fields)
```

Comment:
definition moved to core: https://github.com/ManageIQ/manageiq/pull/17621/files#diff-be0179ddb233d59fa266cfa25e271345R59

Reply:
:+1: thanks @slemrmartin